aries-site/src/layouts/main/Footer.js

```diff
@@ -30,29 +36,36 @@ export const Footer = () => {
         horizontal: size !== 'small' ? 'xlarge' : 'large',
       }}
     >
-      <Box direction="row-responsive">
-        <ThemeModeToggle />
-      </Box>
       <Box direction="row-responsive" gap="medium" align="center">
         <Box>
-          <Text size="small">&copy; {year} Hewlett Packard Enterprise</Text>
+          <Text size="medium">&copy; {year} Hewlett Packard Enterprise</Text>
         </Box>
-        {externalFooterLinks.map((link, index) => (
-          <FooterLink
-            size="small"
-            key={index}
-            href={link.href}
-            label={link.label}
-            target="_blank"
-            rel="noreferrer noopener"
-          />
+        {externalFooterLinks.map(item => (
+          <Link
+            key={item.name}
+            href={item.href}
+            passHref
+          >
+            <NavButton
+              item={item.name}
+              active={router.pathname === (item.href)}
+              target="_blank"
+              href={item.href}
```

Comment:
Since we are passing the Href above, I'm not sure that we need href on the NavButton anymore. Can you try locally and see if it still works properly without the href here?

Reply:
lol I have no idea why this was here! my bad its fine just above